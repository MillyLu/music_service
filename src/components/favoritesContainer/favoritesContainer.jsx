



import Bar from '../bar/bar';
import { FavoritesCenterBlock } from '../favoritesCenterblock/favoritesCenterblock';
import MainSideBar from '../sidebar/sidebar';
import Navigation from '../navigation/nav';
import { Footer } from '../footer/footer';
import * as Styled from './styles';





function FavoritesMain() {
    return(
        <Styled.Main>
            <Navigation />
            <FavoritesCenterBlock />
            <MainSideBar />
        </Styled.Main>
    )
}

export function FavoritesContainer() {
    return(
        <Styled.Container>
            <FavoritesMain />
            <Bar />
            <Footer />
        </Styled.Container>
    )
}

