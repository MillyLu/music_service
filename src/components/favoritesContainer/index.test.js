/* eslint-disable no-undef */
/* eslint-disable no-unused-expressions */
import * as ShallowRenderer from 'react-shallow-renderer';
import { FavoritesContainer } from "./favoritesContainer";


describe('<FavoritesContainer/>', () => {
    it('should render successfully and match the snapshot', () => {
        const shallowRenderer = ShallowRenderer.createRenderer();
        const renderer = shallowRenderer;
        renderer.render(<FavoritesContainer />);
        const result = renderer.getRenderOutput();
        expect(result).toMatchSnapshot();
    })
})