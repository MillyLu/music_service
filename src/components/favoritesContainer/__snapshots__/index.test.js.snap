// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesContainer/> should render successfully and match the snapshot 1`] = `
<styled.div>
  <FavoritesMain />
  <Bar />
  <Footer />
</styled.div>
`;
